### DISCUSSION TITLE
component  distory

### DISCUSSION THREAD

**Discussion by a524313390** [facebookexperimental/Recoil#1786] [Q&A]
https://codesandbox.io/s/agitated-snowflake-hgpd0v?file=/src/pages/About/index.tsx
组件注销初始化atom没有触发。导致每次路由切换控制台会打印之前的数据

**Comment by drarmstr** [comment]
Google translate: Component logout initialization atom did not fire. Causes the console to print the previous data every time the route switches

Not sure I understand the question.  Looking at the example It seems like it prints "Home" on the home page and "About" on the about page.
